repository-data/application/src/main/resources/hcm-config/configuration/users/sitewriter.yaml

```diff
@@ -0,0 +1,8 @@
+definitions:
```

Comment:
@smanivasag I think we need such permission setting for application scope instead of development scope. Because we need these setting applied to both dev, stage, prod.

Reply:
Ah OK, thanks 👍